docs/source/github.md

```diff
@@ -39,3 +39,33 @@ To use this expanded user information, you will need to subclass your
 current spawner and modify the subclass to read these fields from
 `auth_state` and then use this information to provision your Notebook or
 Lab user.
+
+## Restricting access
+
+#### Organisations
+
+If you would like to restrict access to members of specific GitHub organisations
+you can pass a list of organisation names to `allowed_organisations`.
+
+For example, the below will ensure that only members of `org_a` or
+`org_b` will be authorized to access.
+
+`c.GitHubOAuthenticator.allowed_organizations = ["org_a", "org_b"]`
+
+#### Teams
+
+It is also possible to restrict access to members of specific teams within
+organisations using the syntax: `<organisation>:<team-name>`.
+
+For example, the below will only allow members of `org_a`, or
+`team_1` in `org_b` access. Members of `org_b` but not `team_1` will be
+unauthorized to access.
+
+`c.GitHubOAuthenticator.allowed_organizations = ["org_a", "org_b:team_1"]`
+
+#### Notes
```

Comment:
```suggestion
### Organisations

If you would like to restrict access to members of specific GitHub organisations
you can pass a list of organisation names to `allowed_organisations`.

For example, the below will ensure that only members of `org_a` or
`org_b` will be authorized to access.

`c.GitHubOAuthenticator.allowed_organizations = ["org_a", "org_b"]`

### Teams

It is also possible to restrict access to members of specific teams within
organisations using the syntax: `<organisation>:<team-name>`.

For example, the below will only allow members of `org_a`, or
`team_1` in `org_b` access. Members of `org_b` but not `team_1` will be
unauthorized to access.

`c.GitHubOAuthenticator.allowed_organizations = ["org_a", "org_b:team_1"]`

### Notes
```

Reply:
We have been using the american spelling "organizations" instead of the UK based spelling "organisations". Since the configuration option use `z`, let's do it consistently.

Reply:
@sgibson91 or @manics who knows English better than me, does this make sense?

Reply:
If this suggestion is accepted, I'd like to go for a merge and a release.

Reply:
Updated 😄